cqrs-postgres/src/db_wrapper.rs

```diff
@@ -0,0 +1,177 @@
+use crate::util::Sequence;
+use cqrs_core::{CqrsError, EventNumber, RawEvent, Since};
+use num_traits::ToPrimitive;
+use postgres::{rows::Row, types::ToSql};
+use r2d2::{Pool, PooledConnection};
+use r2d2_postgres::PostgresConnectionManager;
+use std::{error, fmt, sync::Arc};
+
+#[derive(Debug, Clone)]
+pub enum DbError {
+    Format(Arc<dyn CqrsError>),
```

Comment:
`std::fmt::Error` is a zero-sized type, so you could just let this be `Format` with no data. However, with the comments from below, we can probably remove this arm altogether.

Reply:
Removed